resume-openfont.cls

```diff
@@ -166,4 +166,4 @@
 % Align Sections
 \let\oldSection\section
 \renewcommand{\section}[1]{\oldSection{#1}\raggedright}
-\renewcommand{\underline}[1]{\uline{#1}}
\ No newline at end of file
+\renewcommand{\underline}[1]{\uline{#1}}
```

Comment:
@stormvirux what's this change here?

Reply:
I did not make any direct change here. Probably might be related to line ending that might have been automatically changed by vs code